server/src/perl/perlimportsWrapper.pl

```diff
@@ -0,0 +1,34 @@
+#!perl
+
+use strict;
+use warnings;
+
+use Try::Tiny qw( catch try );
+
+if ( !eval { require App::perlimports::CLI; 1 } ) {
+    print "\nUnable to run perlimports as it is not installed\n";
+    exit(0);
```

Comment:
Needs to be exit(1) or else the entire tidied file gets replaced with this message.

Reply:
That makes sense.